.travis.yml

```diff
@@ -90,6 +90,8 @@ before_script:
   - mysql -uroot -e 'create database ngx_test; grant all on ngx_test.* to "ngx_test"@"%" identified by "ngx_test"; flush privileges;'
 
 script:
+  - sudo iptables -I OUTPUT 1 -p udp --dport 10086 -j REJECT || exit 0
```

Comment:
We'd better avoid requiring the developers to mess up with the local firewall settings just to run your test case. But it looks tricky to make that code path get exercised without yielding.